README.md

```diff
@@ -94,140 +98,89 @@ $apiContext = ApiContext::create(
 ```
 
 #### Safety considerations
-The file storing the context details (i.e. `bunq.conf`) is a key to your account. Anyone having access to it is able to
-perform any Public API actions with your account. Therefore, we recommend choosing a truly safe place to store it.
+The file storing the context details (i.e. `bunq.conf`) is a key to your account. Anyone having access to it is able to perform any Public API actions with your account. Therefore, we recommend choosing a truly safe place to store it.
+
+If you rather save the context in a database, you can use the `fromJson()` and `toJson()` methods. 
 
 ### Making API calls
 There is a class for each endpoint. Each class has functions for each supported action. These actions can be
 `create`, `get`, `update`, `delete` and `listing`.
 
-Sometimes API calls have dependencies, for instance `MonetaryAccount`. Making changes to a monetary account 
-always also needs a reference to a `User`. These dependencies are required as arguments when performing API calls.
+Before you can start making calls, you must ensure that you have create an ApiContext and loaded in into BunqContext as shown in the examples above. 
+
+The SDK will take care of your user Id, as this id wil never change per ApiContext. The SDK also uses your first active monetary account as primary monetary account. This is alsmot always the same as your billing account. This means, if you do not explicitly pass a monetary account id, the SDK will use the billing account id.  
```

Comment:
`alsmot.`

`This means, if you do not explicitly pass a monetary account id, the SDK will use the billing account id.`
➡️ 
`This means that when you do not explicitly pass a Monetary Account ID, the SDK will use the Monetary Account ID of your billing account.`